src/classes/AffiliationMappingsHealthCheckVMapper.cls

```diff
@@ -198,6 +257,325 @@ public virtual with sharing class AffiliationMappingsHealthCheckVMapper {
         );
     }
 
+   /**********************************************************************************
+    * @description Retrieve a HealthCheckGroup view model reprenting whether enrollment
+    * statuses specified in affiliation mappings correspond to valid picklist field values
+    * listed in the Status picklist field on Affiliation__c.
+    * @return A HealthCheckGroupVModel
+    **********************************************************************************/ 
+    @TestVisible
+    private virtual HealthCheckGroupVModel getActiveExistingStatusHealthCheckGroupVModel(
+        AffiliationMappingsModel affiliationMappingsModel,
+        Map<String, PicklistEntryModel> picklistEntryModelByName
+    ) {
+        List<HealthCheckItemVModel> healthCheckItemsList = new List<HealthCheckItemVModel>();
+        String healthCheckGroupLabel = Label.stgHealthCheckAffiliationMappings;
+
+        HealthCheckGroupVModel healthCheckGroupVModel = new HealthCheckGroupVModel(healthCheckGroupLabel,
+                                                                                 Label.stgHealthCheckStatusPassed,
+                                                                                 0,
+                                                                                 0,
+                                                                                 new List<String>(),
+                                                                                 healthCheckItemsList);
+
+        if (affiliationMappingsModel.autoProgramEnrollment != true) {
+            return healthCheckGroupVModel;
+        }
+
+        if (affiliationMappingsModel.autoProgramEnrollmentStatus == '""') {
+            // Auto-Enrollment Status not specified
+            healthCheckGroupVModel.healthCheckItemList.add(new HealthCheckItemVModel(
+                '05',
+                Label.stgHCAfflMapAutoEnrollStatusTitle,
+                'Fail',
+                Label.stgHealthCheckStatusFailed,
+                String.format(
+                    Label.stgHCAfflMapAutoEnrollStatusNotFound, 
+                    new List<Object>{affiliationMappingsModel.autoProgramEnrollmentStatus}
+                ),
+                Label.stgHCAfflMapAutoEnrollStatusNotFoundFix, 
+                new List<HealthCheckItemVModel>()
+            ));
+
+            healthCheckGroupVModel.totalChecks++;
+            healthCheckGroupVModel.updateHealthCheckStatus();
+
+            return healthCheckGroupVModel;
+        }
+
+        if (picklistEntryModelByName.keySet().contains(affiliationMappingsModel.autoProgramEnrollmentStatus) == false) {
```

Comment:
This should evaluate to the exact same situation. We don't need to perform an extra check here.
```suggestion
        if (picklistEntryModelByName.keySet().contains(affiliationMappingsModel.autoProgramEnrollmentStatus) == false) {
```